doc/building.md

```diff
@@ -1,4 +1,57 @@
-Building nyan
+Building the project
 =============
 
-no code yet :)
+Dependencies
+-------------
+
+```
+gcc >= 5.3.0 or clang >= 3.4
+make >= 3.81
+automake >= 1.14
+autoconf >= 2.69
+libtool >= 2.4.2
```

Comment:
i think most of those can be a much older version, dunno how to test that though. especially `make` can be old i guess.